### PR TITLE
Issue 6298: pravega version change for 0.10 branch

### LINKED ISSUE #6298: Change `pravegaVersion` in gradle.properties for r0.10 branch
**Describe the change**
Change `pravegaVersion` in `gradle.properties` for r0.10 branch 
This is required as part of the release process for Pravega 0.10.

### PR DESCRIPTION
**Pull request by pbelgundi**
Signed-off-by: pbelgundi <prajakta.belgundi@emc.com>

**Change log description**  
Update pravegaVersion for 0.10 branch

**Purpose of the change**  
Fixes #6298

**How to verify it**  
All tests should pass

### CODE REVIEW

**Review by shrids — APPROVED** [review]

**Comment by RaulGracia** [comment]
@pbelgundi I think that we first need to set pravegaVersion to 0.10.0 to make the release at this point, and only after that, change again the suffix to `SNAPSHOT`.

**Review by RaulGracia — CHANGES_REQUESTED** [review]
I think this one comes after the actual release.

**Comment by pbelgundi** [comment]
> I think this one comes after the actual release.

I have done this per instructions in the release wiki: 
https://github.com/pravega/pravega/wiki/How-to-release#major-release

I also see the same kind of version change being made on earlier release branches.

**Comment by RaulGracia** [comment]
@pbelgundi this it what I read:
> In r0.3, create an issue and corresponding pull request to change the pravegaVersion in gradle.properties to 0.3.0. Once that change is merged, we need to tag the commit point in the same way we described for a bug-fix release. See instructions in the previous section.

In this case, we are in `r0.10`, so we should be changing `pravegaVersion=0.10.0`, right?

File: gradle.properties [hunk 1]
```diff
@@ -69,7 +69,7 @@ jjwtVersion=0.9.1
 bouncyCastleVersion=1.69
 
 # Version and base tags can be overridden at build time
-pravegaVersion=0.10.0-SNAPSHOT
+pravegaVersion=0.10.1-SNAPSHOT
```
Comment (andreipaduroiu) [inline gradle.properties:72]: @pbelgundi We first need to set this to `0.10.0`, do the release, then change it to `0.10.1` after the release is done. That way the build artifacts point to a commit in GH that has the correct version set.

**Review by RaulGracia — APPROVED** [review]

File: gradle.properties (hunk 1 above)
Comment (pbelgundi) [reply gradle.properties]: Ok done

**Review by shrids — APPROVED** [review]
Thanks for pointing that out @RaulGracia

**Pull request merged**